docs/data-sources/direct_connect_v2.md

```diff
@@ -0,0 +1,64 @@
+---
```

Comment:
Please use https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/blob/devel/docs/resources/waf_dedicated_cc_rule_v1.md as template, let's keep old aws-style doc structure but with more details for arguments

Reply:
Also I think we don't need to create another bicycle, just look at any hashicorp official provider documentation and lets just keep his structure, because it's familiar for all users.